OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -71,6 +71,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Maximum offset at the maximum range")]
 		public readonly WDist Inaccuracy = WDist.Zero;
 
+		[Desc("Is inaccuracy disabled when sucessfully locked onto target.")]
+		public readonly bool DisableInaccuracyOnLockOn = false;
```

Comment:
Would there be any value on implementing this as `public readonly WDist LockOnInaccuracy = WDist.Zero;` instead of simply toggling between `Inaccuracy` and 0?

Reply:
Good idea, actually.